chucks-pick3-ws/src/main/resources/application-aws.properties

```diff
@@ -9,7 +9,4 @@ spring.jpa.show-sql=true
 spring.jpa.properties.hibernate.format_sql=true
 spring.datasource.hikari.connectionTimeout=30000
 spring.datasource.hikari.idleTimeout=600000
-spring.datasource.hikari.maxLifetime=1800000
-
-chucks_pick3_aws_access_key: AKIAV5HWKAZHMQ2FQF6N
```

Comment:
Are we sure we want to remove the AWS access key now.  I think if we remove it will break the deployment.  Please put revert back and put these changes into the feature branch that deals with the encrypting sensitive information in application.properties.